src/main.cpp

```diff
@@ -564,4 +566,5 @@ void exit_handler(int s)
 
         exit( exit_status );
     }
+    inp_mngr.set_timeout( old_timeout );
```

Comment:
Strikes me as potentially dangerous. `refresh_all` accesses many things, some may not exist while the game is still in the main menu. It also refreshes only the main terrain window, which is pointless if another window (e.g. overmap, inventory, ...) is currently open.